scripts/gen_users_names.pl

```diff
@@ -50,8 +50,8 @@
 		print lc($user_ref->{email}) . "\t$userid\t" . $user_ref->{name} . "\n";
 	}
 
-	if ($user_ref->{twitter} ne '') {
-		#		print "\@" . $user_ref->{twitter} . " ";
+	if ($user_ref->{x} ne '') {
```

Comment:
we have a field named "twitter" for user data, we can't change it like this, unless we also migrate all the existing users. I suggest not to do it at this point, and keep "twitter" as the field name internally.